src/config/mod.rs

```diff
@@ -10,16 +10,21 @@ use serde::Deserialize;
 use tracing::{instrument, trace_span};
 
 use crate::{
+    environment::Environment,
     rule::{InstantiatedRule, Rule, RuleOptions},
     Plugin,
 };
 
 mod config_file;
-pub use config_file::{find_config_file, load_config_file, ParsedConfigFile};
+pub use config_file::{
+    find_config_file, load_config_file, ParsedConfigFile, TreeSitterLintDependencySpec,
+};
 
 #[derive(Builder, Debug, Default, Parser)]
 #[builder(default, setter(into, strip_option))]
 pub struct Args {
+    pub paths: Vec<PathBuf>,
```

Comment:
I guess exposing the ability to run linting against only certain files via the command line